src/Components/CardContainer/Card/Card.js

```diff
@@ -55,7 +74,7 @@ export class Card extends Component {
           <Link to={`/${id}`}>
             <span>{ title }</span>
           </Link>
-          <button onClick={ () => toggleFavorite(id, this.state.favorite) }>FAVE</button>
+          <button className={faveClass} onClick={this.toggleFavorite}>FAVE</button>
```

Comment:
I like what you did here to be able to add some styling to this button so the user knows it was toggled